features/cascading_replication.feature

```diff
@@ -8,6 +8,7 @@ Scenario: check a base backup and streaming replication from a replica
 	And replication works from postgres0 to postgres1 after 20 seconds
 	And I create label with "postgres0" in postgres0 data directory
 	And I create label with "postgres1" in postgres1 data directory
+	And postgres1 has state=running in dcs after 12 seconds
```

Comment:
> How comes it didn't have state=running in DCS?

Because touch_member is not called immediately after starting up postgres.
clone && start takes 2-3 seconds, but loop_wait is 10 seconds, so member would be updated only on the next iteration.